src/httpclient.js

```diff
@@ -5,7 +5,7 @@ const DEFAULT_USER_AGENT = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKi
 class HttpClient {
 
   async init(userAgent = DEFAULT_USER_AGENT) {
-    this.browser = await puppeteer.launch({headless: true});
+    this.browser = await puppeteer.launch({headless: true, args: ['--no-sandbox', '--disable-setuid-sandbox']});
```

Comment:
Please add a *useSandbox* variable as a first argument to *init* with default to *false*. If *false*, set those sandbox args, otherwise the array is empty. On a Linux VPS I had to disable sandbox but on my Windows desktop it works fine. 

Although using sandbox is better for security reasons, I don't think it is much of an issue here. It should be possible to enable/disable, just like changing user agent is supported.

Will also add a link to [troubleshoot](https://github.com/puppeteer/puppeteer/blob/master/docs/troubleshooting.md) problems with puppeteer dependencies on the readme, since I had to install some packages.

Reply:
Hi, Thank you for your advice. I have done the change.